File: engine.py
pointer=-1
lock=0
def setSeq(data) :
    if lock == 1:
        exit(2)
    globals()['pointer']=-1
    globals()['data']=data.split(',')
    globals()['lock']=1

def raw_input() :
    globals()['pointer']+=1
    try:
       return data[pointer]
    except IndexError:
       exit(0)

def open(path,mode) :
     exit(3)
 

File: engine.c
int pointer=-1;
int lock=0;
char *data[100];

int setSeq(char *d)
{
   if(lock == 1){
       exit(2);
   }
   pointer=-1;
   lock=1;
   //TODO
}

int scanf(const char *format, ...){
   //TODO
}

int fprintf(FILE *stream, const char *format, ...)
{
    return 0;
}

int printf(const char *format, ...)
{
    return 0;
}
